card/lib/generators/card/USAGE

```diff
@@ -0,0 +1,21 @@
+Description
+
+Examples
+  wagn generate card new cards pre_fix
```

Comment:
Hey man,

A lot here to respond to, but the main thing I want to say is that I think we're very close to a great solution, and it's all thanks to you.   I just moved some files around, but we're retaining all your major restructuring.

> I'm not that worried about getting an engine to work in a simple application started from scratch. I'm concerned about my use case of an enormous rails app that I wan't to add cards to. 

I know we can do it, but I really want to build one so we can agree upon the endpoint.  Imo, it's weird that we're getting stuck generators; we're debating the mold when we haven't made the statue.  On that level, it seems like the CLI is the *only* part we're ready to clarify.

> If I try to present this as a solution to my colleagues when the basic plumbing isn't right, I'll be sunk from the start. They won't care if the CLI is natural, but about whether the steps work and don't break things.

I hear you.  I don't think I really changed the plumbing that much other than gem organization, right?  As mentioned over there, that's the part that's important to me.  We can now do a LOT of wagn -> decko renaming within the wagn gem.  Then, when it's time to pull the switch, the gem rename should be a snap.

> To me, decko-rails is a shim we need to not have to re-brand all at once.

That's completely different from my conception. The way you described it in the other gem was much better aligned.  Wagn/Decko is everything you need for an app deck.  Decko-Rails is a (very thin) layer wrapped around that with support for making it so that it's easy to plug decks into rails apps.  As you say, there currently is almost nothing to that.

As I understand things, decko-sinatra would be used to plug decko into a sinatra app.  It would not change the internal workings of decko (which, for now at least, would still be based on rails).  If you wanted to build a decko-like app using sinatra for the innards, that would be a different application.

This is what I'm trying to emphasis.  Decko is decko.  There's only one of them.  It may be able to plug into multiple frameworks, but that plugging-in won't hollow out its innards.  The fact that it is currently rails makes it very easy to make it rails compatible.  But it should never depend on decko-rails, which is the mechanism for deploying it as an engine within a host app.